Include/XA/kzla/ksmo.txi

```diff
@@ -0,0 +1,22 @@
+// by Alexander Behrouzi-Shad :)
+
+TOWER;KSMO;N034.01.10.747;W118.26.55.995;
+NE HANGARS/PARKING;KSMO;N034.01.15.383;W118.26.54.725;
+HANGARS;KSMO;N034.00.45.866;W188.27.11.903;
+HANGARS;KSMO;N034.00.48.138;W188.27.07.862;
+FBO;KSMO;N034.00.59.965;W118.27.10.708;
+TRANSIENT PARKING;KSMO;N034.01.10.212;W118.26.44.276;
+
+A;KSMO;N034.00.47.065;W118.27.20.895;
+A;KSMO;N034.00.59.041;W118.27.06.737;
+A;KSMO;N034.01.07.594;W118.26.56.689;
+A1;KSMO;N034.00.44.363;W118.27.22.830;
+A2;KSMO;N034.00.48.958;W118.27.17.367;
+A4;KSMO;N034.01.08.015;W118.26.54.793;
+
+B;KSMO;N034.00.43.791;W118.27.17.347;
+B;KSMO;N034.00.55.922;W118.27.02.333;
+B;KSMO;N034.01.04.544;W118.26.52.208;
+B1;KSMO;N034.00.42.284;W118.27.20.217;
+B2;KSMO;N034.00.46.763;W118.27.14.680;
+B4;KSMO;N034.01.05.875;W118.26.52.209;
```

Comment:
B3 & B5 also seem to be missing. Can you put the labels in for where the graphics should be when #51 gets finished?